src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -366,4 +366,25 @@
   <data name="net_http_feature_UWPClientCertSupportRequiresCertInPersonalCertificateStore" xml:space="preserve">
     <value>Client certificate was not found in the personal (\"MY\") certificate store. In UWP, client certificates are only supported if they have been added to that certificate store.</value>
   </data>
+  <data name="net_http_content_unexpected_end" xml:space="preserve">
+    <value>Encountered an unexpected end of the content stream.</value>
+  </data>
+  <data name="net_http_headers_missing_location" xml:space="preserve">
+    <value>The response code indicates a redirect, but the 'Location' header is missing.</value>
+  </data>
+  <data name="net_http_invalid_chunk_size" xml:space="preserve">
+    <value>The response stream contains an invalid chunk size.</value>
+  </data>
+  <data name="net_http_max_redirects" xml:space="preserve">
+    <value>The maximum number of redirects was exceeded.</value>
+  </data>
+  <data name="net_http_proxy_invalid_scheme" xml:space="preserve">
+    <value>Invalid scheme for proxy. Scheme: '{0}'.</value>
+  </data>
+  <data name="net_http_request_invalid_char_encoding" xml:space="preserve">
+    <value>Requests must not contain non-ASCII characters.</value>
```

Comment:
Can we get rid of the double negative? e.g. "Request headers must contain only ASCII characters."